File: ExampleApp/app/src/main/java/com/motiv/example/PostDetailsActivityContract.java
package com.motiv.example;

import androidx.fragment.app.*;
import dagger.*;
import dagger.android.*;
import dagger.android.support.*;
import javax.inject.*;

public interface PostDetailsActivityContract {

    public interface View {

        public void textview10setText(java.lang.CharSequence arg0);

        public void textview11setText(java.lang.CharSequence arg0);
    }

    public interface Presenter {

        public void daoRepositoryloadPost(java.lang.String arg0);
    }
}

File: ExampleApp/app/src/main/java/com/motiv/example/PostDetailsActivityContract.kt
 
package com.motiv.example

public interface PostDetailsActivityContract {

    public interface View {

        public fun textview10setText(arg0: CharSequence): Unit
        public fun textview11setText(arg0: CharSequence): Unit
    }

    public interface Presenter {

        public fun daoRepositoryloadPost(arg0: String): Unit
    }
}
